src/pages/data-reporting/reports-api/v2.0/index.mdx

```diff
@@ -25,6 +25,22 @@ You can access all the request from the Reports API v2 in the Postman Collection
 
 <Section>
 
+## Sample App
+
+### Progress App
+
+Progress is a simple app that creates reports based on data from chats. It can display the daily, weekly, monthly, and annual statistics. The App indirectly uses **Reports API 2.0** endpoints:
```

Comment:
The problem with this app is that it uses our Reports API v2.0 indirectly. It also uses indirect endpoints made by Dominik. When you check devtools, you will not see, e.g., `/reports/chats/response_time` but `https://us-central1-livechat-experiments.cloudfunctions.net/restApi/responses'  <- endpoint the indirect one API from Dominika. The question is if, in that case, it is okay to leave the description like that or maybe add something?🤔

Reply:
What does it mean indirectly? 🤔

Reply:
We could also mention what authorization method this app implements

Reply:
Wojtek gave me info that this app does not use 3 endpoints mentioned by me below from the docs directly. There is also an intermediate API created by Dominik Sławkowski. When developers check the dev tools, they will not see `/reports/chats/response_time`; they will see `https://us-central1-livechat-experiments.cloudfunctions.net/restApi/responses`  <- endpoint belonging to the intermediate API created by Dominik, and we do not have access to it.
So I am a bit stuck on how to describe it🤔  Or not mention this 'indirect part' at all?🤔 Or maybe - ...The app uses Reports API 2.0 endpoints below together with a supportive, intermediate API:

Reply:
Hm, I wouldn't mention this intermediate API, I guess. When they look in the code, they'll see the endpoints so they should be fine